zookeys.csl

```diff
@@ -47,7 +47,7 @@
   </macro>
   <macro name="author-short">
     <names variable="author">
-      <name form="short" delimiter=" " and="text" delimiter-precedes-last="never" et-al-min="3" et-al-use-first="1" initialize-with=". "/>
+      <name form="short" delimiter=" " and="text" delimiter-precedes-last="never" et-al-min="3" et-al-use-first="1" et-al-subsequent-min="3" et-al-subsequent-use-first="1" initialize-with=". "/>
```

Comment:
I'd much rather you remove the et al settings here entirely and instead include them in the citation line (line 154) below. There's no reason to have the useless settings in there and, as you've seen, it can cause all types of problems when working with styles.